src/components/AllMovies.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+    <section>
+        <BiggestHeader text="All the movies" theme="header--black"/>
+        <div>
+            <CustomInput
+                v-model="search"
+                label="search"
+                placeholder="What are you looking for?"
+                type="search">
+            </CustomInput>
+                    
+            <SelectInput
+                v-model="selectedOption"
+                placeholder="All categories"
+                label="category"
+                :selectOptions="selectOptions">
+            </SelectInput>
+        </div>
+    </section>
+</template>
+
+<script>
+import BiggestHeader from '@/components/BiggestHeader.vue';
+import CustomInput from '@/components/CustomInput.vue';
+import SelectInput from "@/components/SelectInput.vue";
+
+export default {
+    components: {
+        BiggestHeader,
+        CustomInput,
+        SelectInput
+    },
+    data() {
+        return {
+            search: '',
+            selectedOption: '',
+            selectOptions: [
```

Comment:
When suggesting renaming, I meant to give them a specific meaning in this particular case, e.g. `selectedCategory`, `categoryOptions`.